python/src/aac/plugins/contributions/contribution_types/definition_validation_contribution.py

```diff
@@ -8,17 +12,20 @@
 @attrs(hash=False)
 class DefinitionValidationContribution:
     """
-    A class that contains all the relevant information to manage and execute definition validation contributions with the validation process.
+    A contribution for validating a definition.
+
+    A class that contains all the relevant information to manage and execute definition validation
+    contributions with the validation process.
 
     Attributes:
-        name: A string with the name of the command argument
-        definition: The AaC definition of the Validator Plugin
-        validation_function: The validation callback function
+        name (str): A string with the name of the command argument.
+        definition (str): The AaC definition of the Validator Plugin.
+        validation_function (Callable): The validation callback function.
     """
 
     name: str = attrib(validator=validators.instance_of(str))
     definition: Definition = attrib(validator=validators.instance_of(Definition))
-    validation_function: callable = attrib(validator=validators.is_callable())
+    validation_function: Callable = attrib()
```

Comment:
The `is_callable` validator was causing issues when implementing our fix for the "only require validator implementations for active plugins" because the check in our validator required seeing if the `validation_function is None` but the `is_callable` validator doesn't allow `None` for a function.

Reply:
Ah, this can be accomplished by wrapping the is_callable call with the optional function like so:
```    validation_function: Optional[Callable] = attrib(validator=validators.optional(validators.is_callable()))``` For some reason I'm unable to create a suggestion for this line.

Reply:
Okay, I'll use that - thanks!